### PR TITLE
probot stale

### PR DESCRIPTION
**Pull request by ahegde3**
issue #64 
I have created a .yml file with suitable configuration to remove stale issues.
I have limited it only to pull request so it wont check for opened issues.
After 15 days of inactivity the pull request becomes stale and after further 7 days it will be closed.

### CODE REVIEW

**Comment by nurdtechie98** [comment]
@inishchith what do you think ??

**Comment by inishchith** [comment]
@ahegde3 Can you please keep only the commits which are relevant to this PR?

**Comment by ahegde3** [comment]
yeah.I will make the change

**Comment by ahegde3** [comment]
I have updated commit history

File: .github/stale.yml [hunk 1]
```diff
@@ -0,0 +1,64 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 15
```
Comment (inishchith) [inline .github/stale.yml]: keep this to 30 days

File: .github/stale.yml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+# Number of days of inactivity before an Issue or Pull Request with the stale label is closed.
+# Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
+daysUntilClose: 7
```
Comment (inishchith) [inline .github/stale.yml]: keep this to 10 days

File: .github/stale.yml [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+# Only issues or pull requests with all of these labels are check if stale. Defaults to `[]` (disabled)
+onlyLabels: []
```
Comment (inishchith) [inline .github/stale.yml]: We aren't closing every other issue.
Only "bugs", "critical", "duplicate", "invalid",  must be checked.

Feel free to suggest some good labels. we can add them in general.

File: .github/stale.yml [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+# Issues or Pull Requests with these labels will never be considered stale. Set to `[]` to disable
+exemptLabels:
```
Comment (inishchith) [inline .github/stale.yml]: I'd remove this as `onlyLabels` is now set.

File: .github/stale.yml [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+exemptProjects: true
+
+# Set to true to ignore issues in a milestone (defaults to false)
+exemptMilestones: false
+
+# Set to true to ignore issues with an assignee (defaults to false)
+exemptAssignees: false
+
+# Label to use when marking as stale
+staleLabel: Stale
```
Comment (inishchith) [inline .github/stale.yml]: change this to `action required`

File: .github/stale.yml [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+# Comment to post when removing the stale label.
+# unmarkComment: >
+#   Your comment here.
+
+# Comment to post when closing a stale Issue or Pull Request.
+# closeComment: >
+#   Your comment here.
+
+# Limit the number of actions per hour, from 1-30. Default is 30
+limitPerRun: 10
```
Comment (inishchith) [inline .github/stale.yml]: keep this to 5

File: .github/stale.yml [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+
+# Limit to only `issues` or `pulls`
+ only: pulls
```
Comment (inishchith) [inline .github/stale.yml]: please remove this line

**Comment by ahegde3** [comment]
I have done the asked changes

**Comment by inishchith** [comment]
@ahegde3 Please remove the comments and update the 1st line with the link to usage section of the Probot repository.

**Comment by ahegde3** [comment]
Have updated as per request

**Pull request merged**